devito/functions/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .constants import *  # noqa
+from .dimension import *  # noqa
```

Comment:
if we put dimension here, and it does make sense to me, I'm not sure calling the subdirectory "functions" makes a lot of sense